src/app/components/admin/featured/featured.component.ts

```diff
@@ -52,15 +52,15 @@ export class FeaturedComponent implements OnInit {
     }
   }
 
-  async getListItems() {
-    const result = await this.featuredService.getListItems(this.parent);
+  async listItemsRef() {
```

Comment:
In this case we are actually doing a one-time get operations. We can keep this named getListItems() as well as any other promise-based operation that gets data one time then stops.